### PR TITLE
Fix friday bug

### PR DESCRIPTION
**Pull request by benjick**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const isWeekend = date => {
   const dateString = date;
   const weekDay = dateString.getDay();
-  const hour = ((dateString.getHours() + 11) % 12 + 1);
-  return ((weekDay === 5 && hour >= 5) || weekDay === 6 || weekDay === 0);
+  const hour = dateString.getHours() + 1;
```
Comment (wibron) [inline index.js:4]: Maybe the `dateString` variable can be removed? Accessing `date` directly seems more straightforward.

File: test.js [hunk 2]
```diff
@@ -5,8 +5,10 @@ const helg = require('./');
 const wednesday = new Date(1414580400000);
 const saturday = new Date(1414839600000);
 const friday = new Date(1416600000000);
+const friday2 = new Date(1474020949000); // 09/16/2016 @ 10:15am (UTC)
```
Comment (wibron) [inline test.js]: Not sure about this naming. What about `fridayBeforeNoon`?

**Review by wibron — COMMENTED** [review]
Great find!

File: test.js (hunk 2 above)
Comment (benjick) [reply test.js]: I like `friday2` more

Comment (benjick) [reply test.js]: Fixed

File: index.js (hunk 1 above)
Comment (benjick) [reply index.js:4]: Not in scope of this PR

**Pull request merged**